pkgs/development/libraries/science/biology/elastix/default.nix

```diff
@@ -11,10 +11,24 @@ stdenv.mkDerivation rec {
     sha256 = "1zrl7rz4lwsx88b2shnl985f3a97lmp4ksbd437h9y0hfjq8l0lj";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "itk-5.1-compat.patch";
+      url = "https://github.com/SuperElastix/elastix/commit/402e9a26f22f805b8f2db00c00e59f75fa1783ad.patch";
+      sha256 = "1il6gc1lgy78i0w6gkkppr61nh6g0yjspbfk19hcz20778m5jhz9";
+    })
+    (fetchpatch {
+      name = "fix-osx-build.patch";
+      url = "https://github.com/SuperElastix/elastix/commit/52e1dc3928046f9fbb85d4b2ecd0d5175fa9695d.patch";
+      sha256 = "1hf7kgx1jv497pf0x5wj79sy1wncxcvhrkix9w086lcr8zwxvn9q";
+    })
+  ];
+
+
   nativeBuildInputs = [ cmake python3 ];
   buildInputs = [ itk ];
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin;  # usual dynamic linker issues
```

Comment:
What do you mean by usual issues? Can we resolve with `preCheck = "export DYLD_LIBRARY_PATH ...`?

Reply:
I tried doing something naive similar to what is done with LD_LIBRARY_PATH already but that didn't work.  I don't really do much development on OS X so don't mind disabling this package if you prefer.